packages/apps/human-app/server/src/modules/job-assignment/job-assignment.service.ts

```diff
@@ -0,0 +1,79 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { lastValueFrom } from 'rxjs';
+import {
+  JobsAssignmentParamsCommand,
+  JobsAssignmentParamsData,
+  JobAssignmentResponse,
+  JobAssignmentCommand,
+  JobAssignmentData,
+  JobsAssignmentResponse,
+} from './interfaces/job-assignment.interface';
+import { HttpService } from '@nestjs/axios';
+import { InjectMapper } from '@automapper/nestjs';
+import { Mapper } from '@automapper/core';
+
+@Injectable()
+export class JobAssignmentService {
+  constructor(
+    public httpService: HttpService,
+    @InjectMapper() private readonly mapper: Mapper,
+  ) {}
+
+  async processJobAssignment(
+    url: string,
+    jobAssignmentCommand: JobAssignmentCommand,
+  ): Promise<JobAssignmentResponse> {
+    const jobAssignmentData = this.mapper.map(
+      jobAssignmentCommand,
+      JobAssignmentCommand,
+      JobAssignmentData,
+    );
+    try {
+      const options = {
+        method: 'POST',
+        url: `${url}/assignment`,
+        data: jobAssignmentData,
+      };
+      const response = await lastValueFrom(this.httpService.request(options));
+      return response.data;
+    } catch (error) {
+      if (error.response) {
+        throw new HttpException(error.response.data, error.response.status);
+      } else {
+        throw new HttpException(
+          'Internal Server Error',
+          HttpStatus.INTERNAL_SERVER_ERROR,
+        );
```

Comment:
merge with main and use exception filter

Reply:
Fixed in: 4ba85d0b